### ISSUE TITLE
IE11 not working 

### ISSUE THREAD

**Issue by brentgees** [rafaelp/css_browser_selector#22]
I tested it for ie11 on windows 7.
But I get a red window instead of a yellow window

**Comment by verbatim** [comment]
I apologize. I have no access to IE10 or IE11. Do you have the user agent string? I have seen these by doing some searches:

IE10:
Mozilla/5.0 (compatible; MSIE 10.0; Windows NT 6.1; Trident/6.0)

IE11
Mozilla/5.0 (IE 11.0; Windows NT 6.3; Trident/7.0; .NET4.0E; .NET4.0C; rv:11.0) like Gecko

**Comment by brentgees** [comment]
No need to apologize, it's alreay great that you share it :+1: 
And I also did some searches and tryed it with user agent string, but that didn't work for me :/

**Comment by verbatim** [comment]
try:
http://www.whatsmyuseragent.com/

or
http://www.useragentstring.com/

**Comment by SoftCreatR** [comment]
https://github.com/WoltLab/WCF/blob/8ac2c327dd34b93db4a3041ba2ca506360fffb17/wcfsetup/install/files/lib/data/user/online/UserOnline.class.php#L169

Written in PHP, but the regular expression itself should also work in JS. It matches all IE versions, including IE11.

**Comment by eviweb** [comment]
Hi,

if someone have time to review this patch ;)

Thanks

**Comment by verbatim** [comment]
Yes, I will take a look. I myself have no access to IE10 or IE11. But I will take a look at this and integrate when I can.
Thanks.

**Comment by wayferer** [comment]
FWIW I have just tested the above commit on my IE11 and it works for me.

**Comment by devanshidns** [comment]
Where is the updated code for ie 11?
I also get the string as:
Mozilla/5.0 (IE 11.0; Windows NT 6.3; Trident/7.0; .NET4.0E; .NET4.0C; rv:11.0) like Gecko

**Comment by wayferer** [comment]
@devanshidns https://github.com/eviweb/css_browser_selector/commit/0ab8629a1b5d0ee8ff13f842b4d6cdcd622e5065

**Comment by hbmueller** [comment]
IE 11 can be tested with:
!(window.ActiveXObject) && "ActiveXObject" in window

http://stackoverflow.com/questions/21825157/internet-explorer-11-detection

**Comment by TheJaredWilcurt** [comment]
This should work for detecting IE6-11 and potentially beyond.
- https://gist.github.com/TheJaredWilcurt/43c6147bc8a940084a61
